test/travis.sh

```diff
@@ -68,16 +63,16 @@ if [ "$RULESETS_CHANGED" ]; then
   if [ "$TEST" == "rules" ]; then
     echo >&2 "Performing comprehensive coverage test."
     docker_build
-    docker run --rm -ti -v $(pwd):/opt httpse python utils/ruleset_filenames_validate.py
-    docker run --rm -ti -v $(pwd):/opt httpse bash -c "utils/validate.sh"
-    docker run --rm -ti -v $(pwd):/opt httpse bash -c "test/rules.sh"
+    docker run --rm -ti -v $(pwd):/opt electronicfrontierfoundation/https-everywhere-docker-base python utils/ruleset_filenames_validate.py
+    docker run --rm -ti -v $(pwd):/opt electronicfrontierfoundation/https-everywhere-docker-base bash utils/validate.sh
```

Comment:
Do we need `bash` here? `bash` can execute scripts by itself if they have `x` bit set.